web/server/api/endpoints/commands.py

```diff
@@ -28,6 +29,7 @@
 @router.post("/apply", response_model=t.Optional[models.PlanApplyStageTracker])
```

Comment:
I haven't tested it, but I think you might be able to do the following:
```suggestion
@router.post("/apply", response_model=t.Optional[models.PlanApplyStageTracker], status_code=HTTP_204_NO_CONTENT)
```

Reply:
giving me error `Status code 204 must not have a response body`